transform/transformers/CSTransformer.py

```diff
@@ -0,0 +1,102 @@
+import zipfile
```

Comment:
CORATransformer inherits from CSTransformer in order to create zipfiles. Overrides other behaviour.